src/components/pickers/ComboBox/ComboBox.tsx

```diff
@@ -294,6 +295,7 @@ function ComboBox<T extends object>(props: CubeComboBoxProps<T>, ref) {
         {suffix}
         {!hideTrigger ? (
           <TriggerElement
+            data-qa="ComboBox_Trigger"
```

Comment:
ComboBoxTrigger? There is no any `underscored` data-qa attributes in whole project.

Reply:
Good point.